src/whisper.cpp

```diff
@@ -5434,6 +5438,11 @@ void whisper_vad_free(whisper_vad_context * ctx) {
             ggml_backend_free(backend);
         }
 
+        ggml_free(ctx->ctx);
```

Comment:
No need to keep the `ggml_context` - you can safely free it after `ggml_backend_alloc_ctx_tensors` in `whisper_vad_init_context()`.